Containerfile.ubuntu20.04

```diff
@@ -1,15 +0,0 @@
-FROM ubuntu:20.04
-
```

Comment:
I'm guessing it was problematic because of golang 1.15?

Reply:
@cfergeau Yes, that's right.

Reply:
Would be useful to have this explicityll in the commit log rather than leaving this as a guess.